src/models/lazy-audio.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * Wrapper for the Audio object (HTMLAudioElement) that supports lazy loading/playing.
+ *
+ * Future: Is it better for this class to compose an Audio, or directly inherit from it?
+ * Compose Audio (Pros):
```

Comment:
If we think we won't need to migrate to howler.js in the future, I think either is fine. But if so, I think we should be composed of an audio file